docs/dev/build_riscv64.md

```diff
@@ -0,0 +1,184 @@
+# Cross compile OpenVINO™ Runtime for RISCV64 systems
+This guide shows how to build OpenVINO Runtime for 64-bit RISC-V devices. Due to limited resources, cross compilation is used now for building OpenVINO targeting RISC-V development boards.
+
+Cross compilation was tested on the following hosts:
+- Ubuntu 22.04 (64-bit), x64
+
+The software was validated on the following devices:
+- [Lichee Pi 4A](https://wiki.sipeed.com/hardware/en/lichee/th1520/lp4a.html) with RVV 0.7.1
+- [Banana Pi BPI-F3](https://www.banana-pi.org/en/banana-pi-sbcs/175.html) with RVV 1.0
+
+
+## Software requirements
+
+- [CMake](https://cmake.org/download/) 3.13 or higher
+- GCC 7.5 or higher (for non-RVV) / [xuantie-gnu-toolchain](https://github.com/XUANTIE-RV/xuantie-gnu-toolchain) (for RVV)
+- Python 3.10 for OpenVINO Runtime Python API
+
+## How to build
+### Build with RVV
+0. Prerequisite - build `xuantie-gnu-toolchain` and `qemu`:
+   ```sh
+   git clone https://github.com/XUANTIE-RV/xuantie-gnu-toolchain.git
+   cd xuantie-gnu-toolchain
+   ./configure --prefix=<xuantie_install_path>
+   make linux build-qemu -j$(nproc)
+   ```
+   > **NOTE**: The `build-qemu` target is optional, as it is used to build the `qemu` simulator. However, it is recommended to build the `qemu` simulator, since it is much more convenient to validate the software on your host than on your devices.
+
+1. Clone OpenVINO repository and init submodules:
+   ```sh
+   git clone https://github.com/openvinotoolkit/openvino.git
+   cd openvino
+   git submodule update --init --recursive
+   ```
+
+2. Install build dependencies using the `install_build_dependencies.sh` script in the
+   project root folder.
+   ```sh
+   sudo ./install_build_dependencies.sh
+   ```
+
+3. Create a build folder:
+   ```sh
+   mkdir build && cd build
+   ``` 
+
+4. To cross compile OpenVINO Runtime for RISC-V devices, run `cmake` with specified `CMAKE_TOOLCHAIN_FILE` and `RISCV_TOOLCHAIN_ROOT`:
+   ```sh
+   cmake .. \
+     -DCMAKE_BUILD_TYPE=Release \
+     -DCMAKE_INSTALL_PREFIX=<openvino_install_path> \
+     -DENABLE_INTEL_CPU=ON \
+     -DENABLE_INTEL_GPU=OFF \
+     -DENABLE_INTEL_NPU=OFF \
+     -DENABLE_AUTO=ON \
+     -DENABLE_HETERO=ON \
+     -DENABLE_MULTI=ON \
+     -DENABLE_JS=OFF \
```

Comment:
I think we can skip these keys since `AUTO`, `HETERO` and `MULTI` are built by default. As for `-DENABLE_JS=OFF`, I suggested you to disable this target just to reduce build time.
In general it should work and not mentioned in guide.
Could you please remove them?

Reply:
Sure!

Reply:
GPU should be disabled on cmake level. Our cmake scripts should have reasonable defaults to avoid such long list of non-default configuration values.

I ideally, I need no options except toolchains to build default OpenVINO for RISC-V

Reply:
So should I set these keys like `-DENABLE_INTEL_CPU=ON -DENABLE_INTEL_GPU=OFF -DENABLE_INTEL_NPU=OFF` inside toolchain files intead of in the command line?

Reply:
we have some default values in cake/features.cmake file. You can adjust it there

Reply:
Thank you for telling me that! I will remove these non-default keys here then!